test/helpers/assertTx.js

```diff
@@ -0,0 +1,26 @@
+function AssertTx() {
+
+    this.run = async (_func, _args) => {
+        var funcRes;
+        try {
+            funcRes = await _func(..._args);
+        } catch(e) {
+            if (e.toString().indexOf("VM Exception while processing transaction: revert") != -1) {
```

Comment:
Replace to `~e.toString().indexOf("VM Exception while processing transaction: revert")`